library/cli.py

```diff
@@ -59,7 +59,7 @@
 - name: return parsed command output
   cli:
     command: show version
-    parser: parsers/show_version.yaml
+    parser: parsers/show_version
```

Comment:
@trishnaguha Not sure if this change I've made is correct or not

Reply:
@gundalow You can leave this as yaml since the engine is text_parser by default.